### PR TITLE
14.0 migrate

### PR DESCRIPTION
**Pull request by sebastienbeau**
Mass migration, WIP

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@simahawk runbot have failed do you have an idea why ?

**Comment by simahawk** [comment]
@sebastienbeau logs are not available :disappointed:  Anyway it's orange and not red. Likely it's only a warning.

**Comment by sebastienbeau** [comment]
@simahawk now it fixed !
@hparfr will take some time for testing the storage_product_image_pos wait for his feedback before merging

**Comment by hparfr** [comment]
storage_image_product_pos migrated in https://github.com/akretion/storage/pull/1

**Comment by sebastienbeau** [comment]
Ready for merging

**Review by hparfr — APPROVED** [review]

**Comment by simahawk** [comment]
@sebastienbeau please clean the commits before :) I'd like to review also ;P

File: requirements.txt
Comment (simahawk) [inline requirements.txt:3]: do we need these deps at all? They will come from external deps w/ MQT=PIP

**Review by simahawk — CHANGES_REQUESTED** [review]
mostly for commits cleanup

File: requirements.txt
Comment (PierrickBrun) [reply requirements.txt:3]: This is autogenerated so this is not really a problem to mainain IMO

**Review by PierrickBrun — APPROVED** [review]
I have modules depending on it that work fine depending on it, also LGTM (apart from the commit cleaning)

File: requirements.txt
Comment (PierrickBrun) [reply requirements.txt:3]: FYI I tested removing it and pre-commit made me put it again

**Comment by PierrickBrun** [comment]
@simahawk I cleaned the commits, could you update your review ? :)

**Review by simahawk — APPROVED** [review]

**Pull request merged**
